ARCs/arc-0004.md

```diff
@@ -183,16 +196,24 @@ The JSON structure for such an object is:
 interface Interface {
   /** A user-friendly name for the interface */
   name: string,
+  /** Optional, user-friendly description for the interface */
+  desc?: string,
   /** All of the methods that the interface contains */
   methods: Method[]
 }
 ```
 
+Interface names **MUST** satisfy the regular expression `[_A-Za-z][A-Za-z0-9_]*`.
+Interface names starting by `ARC` are reserved to interfaces defined in ARC.
+Interfaces defined in `ARC-XXXX` (where `XXXX` is a 0-padded number) **SHOULD**
+start with `ARC-XXXX`.
```

Comment:
```suggestion
start with `ARC_XXXX`.
```
The hyphen character is not accepted by the above regex, so probably an underscore should be used instead.